packages/main/src/components/ThemeProvider/index.tsx

```diff
@@ -79,6 +91,26 @@ const ThemeProvider: FC<ThemeProviderPropTypes> = (props: ThemeProviderPropTypes
     };
   }, []);
 
+  useEffect(() => {
+    if (_versionInfoInjected) {
+      return () => {
+        // noop
+      };
```

Comment:
Why do we need a cleanup function here? Couldn't we just use `return;`?

Reply:
just for the sake of having a cleanup function, I was under the impression that an effect should either always or never return a cleanup function.